packages/store/src/standalone-features/root-providers.ts

```diff
@@ -1,15 +1,9 @@
 import { APP_BOOTSTRAP_LISTENER, Provider, inject } from '@angular/core';
-import {
-  ɵStateClass,
-  ɵNGXS_STATE_CONTEXT_FACTORY,
-  ɵNGXS_STATE_FACTORY,
```

Comment:
@arturovt  `ɵNGXS_STATE_FACTORY` is used by `@angular-ru/ngxs` ☹

Reply:
They will need to use the `ɵprovideNgxsInternalStateTokens` in their providers list.

Reply:
You can still inject the factory through the token.

Reply:
Through which token?

Reply:
I can’t find any docs regarding angular-ru/ngxs, it’s all empty. Is there any link?

Reply:
What if we expose `StateFactory` as `ɵɵStateFactory` from the `@ngxs/store`? Those tokens will be removed since you'll then might be able to inject the `ɵɵStateFactory` directly.

Reply:
The is not about providing the factory. The library is using and injecting `ɵNGXS_STATE_FACTORY`:
https://github.com/Angular-RU/angular-ru-sdk/blob/845f53c08d352dc67d953446bb15daba37cf02e2/libs/ngxs/internals/services/ngxs-data-injector.service.ts#L22

How do you inject it without having access to the token?

Reply:
I don't understand the phrase "having access to the token." Why don't you have access to the token?

Reply:
ngxs no longer exports `ɵNGXS_STATE_FACTORY` and `ɵNGXS_STATE_CONTEXT_FACTORY` tokens after this PR was merged.

Reply:
Ok, my bad. For some reason instead of `import`, I thought `export` was removed. The tokens are still exported from `@ngxs/store/internals`. I'll try to use `ɵprovideNgxsInternalStateTokens()` to see if it fixes the issue.

Reply:
`ɵprovideNgxsInternalStateTokens()` fixes the issue. Sorry for the confusion.